images/aws-scripts/create-eks-cluster.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to build an EKS cluste from our argo workflow
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+EKS_CLUSTER_NAME="${EKS_CLUSTER_NAME}"
+
+# Create EKS Cluster
+# AI: Need to determine which NG template we need????
+eksctl create cluster \
```

Comment:
It would be good to have a cluster template. It's much easier to update and customize the cluster later